core/templates/calendar.html

```diff
@@ -1,29 +1,98 @@
-<!--<!DOCTYPE html>
-<html>-->
-    <link rel="stylesheet" type="text/css" href="\css\calendar.css">
+<!DOCTYPE html>

```

Comment:
These template files are subsumed by the `layout.html` file which already includes some of these tags. Just to be safe, you might remove the doctype, as well as the `html` and `body` opening and closing tags.